models/articles.model.js

```diff
@@ -92,3 +92,12 @@ WHERE  article_id = $2 returning *;
     return rows[0];
   });
 };
+
+exports.selectArticleQueries = (topic, sort_by, order) => {
+  return this.selectArticles(sort_by, order).then((articles) => {
+    const queryTopics = articles.filter(
```

Comment:
We should absolutely not be using a filter to get articles by topic. This should be handled in your query. This function is not needed and the query string to the database should make use of conditional logic to handle your sorts and order bys